src/test/java/myfirstmodule/MendixUnitTestBase.java

```diff
@@ -145,19 +146,24 @@ protected static void mockInstantiatingProxyClasses() {
 		}
 
 		@SuppressWarnings("unchecked")
-		var mxruntime = (Map<String, Map<String, Object>>) m2ee.get("mxruntime");
-		MF_CONSTANTS = mxruntime.get("MicroflowConstants");
+		var mxruntime = (Map<String, Object>) m2ee.get("mxruntime");
+		@SuppressWarnings("unchecked")
+		var constants = (Map<String, Object>) mxruntime.get("MicroflowConstants");
+		MF_CONSTANTS = constants;
```

Comment:
Could you explain why we need to split this assignment into two?

Reply:
This is the only way I was able to suppress the unchecked cast exception. If you know how to work around that, please let me know.

Reply:
I get it, it's fine then.